road/validation_road.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pandas as pd
+from simpledbf import Dbf5
+import configparser
+import json
+import yaml
+from sklearn.linear_model import LinearRegression
+from io import StringIO
+import geopandas as gpd
+from scatter import *
+
+
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+# Extract the CHAMP input file names
+dbf_directory = config['DBF']['DBF_Directory']
+dbf_files = [dbf_file.strip() for dbf_file in config['DBF']['DBF_Files'].split(',')]
+dbf_column_names = [name.strip() for name in config['DBF']['DBF_Column_Names'].split(',')]
+
+# Extract the observed file name and tab
+excel_file_path = config['OBSERVED FILE']['Excel_File_Path']
+sheet_name = config['OBSERVED FILE']['Sheet_Name']
+extra_columns = [col.strip() for col in config['OBSERVED FILE']['Excel_Extra_Columns'].split(',')]
+obs_usecols = config['OBSERVED FILE']['Obs_usecols']
+time_periods = ['AM', 'MD', 'PM', 'EV', 'EA']
+
+#Part 1 
+# Scatter variables
+chosen_timeperiod = config['SCATTER INPUT']['Chosen_period']
+classification_col = config['SCATTER INPUT']['Classification_col']
+combined_df_cols = [col.strip() for col in config['SCATTER INPUT']['Combined_DF_Cols'].split(',')]
+
+# Est-Obs Plot
+fields1 = config['EST SCATTER PLOT']['Fields'].split(', ')
+nominal_fields1 = config['EST SCATTER PLOT']['NominalFields'].split(', ')
+x_field1 = config['EST SCATTER PLOT']['XField']
+y_field1 = config['EST SCATTER PLOT']['YField']
+name1 = config['EST SCATTER PLOT']['Name']
+
+# Percentile Plot
+fields2 = config['Perc SCATTER PLOT']['Fields'].split(', ')
+nominal_fields2 = config['Perc SCATTER PLOT']['NominalFields'].split(', ')
+x_field2 = config['Perc SCATTER PLOT']['XField']
+y_field2 = config['Perc SCATTER PLOT']['YField']
+name2 = config['Perc SCATTER PLOT']['Name']
+
+dashboard_num = config['YAML']['Dashboard_number']
+
+
+
+def handle_merged_header(excel_file, sheet_name, extra_columns):
+    df = pd.read_excel(excel_file, sheet_name=sheet_name, header=None, skiprows=1)
+    header = df.iloc[0]
+    df = df[1:]
+    df.columns = header

```

Comment:
should this be specified in the read_excel instead of setting it after having read the file?